llvm/lib/Target/AMDGPU/AMDGPUTargetMachine.cpp

```diff
@@ -34,6 +34,7 @@
 #include "R600.h"
 #include "R600TargetMachine.h"
 #include "SIFixSGPRCopies.h"
+#include "SIFoldOperands.h"
```

Comment:
Is this required ?

Reply:
Yes, it is used by the macros in `AMDGPUPassRegistry.def`.